app/src/androidTest/java/com/github/h3lp3rs/h3lp/H3lpAppTest.kt

```diff
@@ -0,0 +1,144 @@
+package com.github.h3lp3rs.h3lp
+
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Intent
+import android.location.Location
+import android.net.Uri
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import com.github.h3lp3rs.h3lp.dataclasses.*
+import com.github.h3lp3rs.h3lp.dataclasses.BloodType.ABn
+import com.github.h3lp3rs.h3lp.dataclasses.Gender.Female
+import com.github.h3lp3rs.h3lp.dataclasses.Gender.Male
+import com.github.h3lp3rs.h3lp.dataclasses.MedicalInformation.Companion.MAX_HEIGHT
+import com.github.h3lp3rs.h3lp.dataclasses.MedicalInformation.Companion.MAX_WEIGHT
+import com.github.h3lp3rs.h3lp.locationmanager.GeneralLocationManager
+import com.github.h3lp3rs.h3lp.locationmanager.LocationManagerInterface
+import com.github.h3lp3rs.h3lp.signin.SignInActivity
+import com.github.h3lp3rs.h3lp.storage.Storages
+import com.github.h3lp3rs.h3lp.storage.Storages.MEDICAL_INFO
+import org.apache.commons.lang3.RandomUtils.nextBoolean
+import org.mockito.Mockito
+import org.mockito.Mockito.`when`
```

Comment:
if you do  import org.mockito.Mockito.\`when\` as When, then you don't need the ugly ` characters when you call When :tada: